docs/conf.py

```diff
@@ -88,55 +79,22 @@
     'stravalib/tests/functional',
     'stravalib/tests/unit',
     'stravalib/tests/resources'
-
 ]
 
-# The reST default role (used for this markup: `text`) to use for all
-# documents.
-#default_role = None
-
-# If true, '()' will be appended to :func: etc. cross-reference text.
-#add_function_parentheses = True
-
-# If true, the current module name will be prepended to all description
-# unit titles (such as .. function::).
-#add_module_names = True
-
-# If true, sectionauthor and moduleauthor directives will be shown in the
-# output. They are ignored by default.
-#show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
-# A list of ignored prefixes for module index sorting.
-#modindex_common_prefix = []
 
-# If true, keep warnings as "system message" paragraphs in the built documents.
-#keep_warnings = False
+# The title in the left hand corner of the docs
+html_title = "Stravalib Docs"
+# Theme and css
+html_theme = "furo"
+html_static_path = ["_static"]
+html_css_files = ["stravalib.css"]
```

Comment:
This just adds the custom css to clean up the look of our docs!

Reply:
Ok, I think the .css sets a fixed font color that creates the issue in dark mode. See my inline comment.

Reply:
@jsamoocha yes great catch. i hadn't tested on dark mode. I just pushed a fix that removes all color settings from the CSS. i keep forgetting about dark mode options in these themes. 

<img width="1350" alt="Screen Shot 2022-11-29 at 8 24 04 AM" src="https://user-images.githubusercontent.com/7649194/204570143-bfdb5b42-48b2-4ecb-bc03-586fcb93e1ac.png">

Reply:
It was pure coincidence that I found that dark mode issue - just decided to try `make -C docs serve` after sunset 😁